modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -9,1064 +9,400 @@ _Enterprise Edition_
-----
--> Mule Agent Unpacked
+_proxy-password_`
+|Specifies the proxy configuration to use when registering with the connection.
 
+See xref:rtm-agent-proxy-config.adoc[Configure the Agent to Connect Through a Proxy Server].
+|===
 
 
-The communication channel for the agent will be encrypted using public/private key certificates.
-In the following steps you are asked to provide the keystore and truststore. Both keystore and
-truststore formats must be JKS.
 
+== Register Mule with the MuleSoft-Managed Runtime Manager
 
-Keystore location (?):/security/rmakeystore.jks
-Keystore Password (?): mulesoft
-Truststore location (?):/security/cacerts.jks
-Keystore Alias (?):rma
-Keystore Alias Password (?): mulesoft
+To register Mule with the MuleSoft-managed Anypoint Platform Runtime Manager console, you need:
 
-        INFO: Mule agent was successfully configured to use a TLS channel for REST communications.
+* A valid registration token, which identifies a specific environment for a specific business group in Anypoint Platform.
+* An instance name, which identifies Mule in the Runtime Manager console.
 
 
-> more ..\conf\mule-agent.yml
+To register Mule:
 
-transports:
-  websocket.transport:
-    enabled: false
-
-  rest.agent.transport:
-    restSecurity:
-      keyStoreFile: clientkeystore.jks
-      keystorePassword: mulesoft
-      keyStoreAlias: client
-      keyStoreAliasPassword: mulesoft
-    port: 9999
-
-services:
-  mule.agent.jmx.publisher.service:
-    enabled: true
-    frequency: 15
-    frequencyTimeUnit: MINUTES
-
-----
+. In Runtime Manager, click *Servers* in the left menu.
+. Click the *Add Server* button.
 +
-This command imports the `/security/cacerts.jks` truststore file into the `$MULE_HOME/conf` folder and renames it  `anypoint-truststore.jks`.
+Runtime Manager generates the `amc_setup` command, including `_token_` and `_server-name_`, to use to register Mule in your environment. 
 +
-[NOTE]
-In Runtime Manager agent version 1.10.0 and later, the name of the default truststore is `anypoint-truststore.jks`. In earlier versions of Runtime Manager agent, the name is `truststore.jks`.
-
-. Restart Mule runtime engine and verify that the Runtime Manager agent REST interface starts successfully.
-. Add SSL debugging to the Mule runtime engine logging:
-+
-`./mule -M-Djavax.net.debug=all`
-
-=== Configure Two-Way TLS for Mule Runtime Engine Managed by Control Plane
-
-To configure two-way TLS when Mule is connected to the control plane:
-
-. Generate a keystore (public-private key pair) to identify the Runtime Manager agent (server).
+The registration token includes your organization's ID and the current environment.
+. Click *Copy* to copy the command.
+. In a terminal window, change to the `$MULE_HOME/bin` directory for the Mule that you're registering.
+. Paste the command on the command line.
+. Change the instance name `server-name` to the unique name to use to label Mule in the Runtime Manager console.
```

Comment:
I would specify that the server-name is unique across environments. You can have the same server name for the same organization in different environments

Reply:
Added.